include/seqan3/alignment/matrix/debug_matrix.hpp

```diff
@@ -156,8 +156,10 @@ class debug_matrix
 
         size_t const _row = !_transpose ? row : col;
         size_t const _col = !_transpose ? col : row;
+        size_t const _mask_row = _transpose == _transpose_mask ? row : col;
```

Comment:
Ah, so if both matrices are transposed (or both not transposed), we don't need to change anything, otherwise we need to transpose one of them?

Reply:
Yeah unfortunately :(